lib/vitalsigns/command.rb

```diff
@@ -2,15 +2,25 @@
 require 'json'
 
 module Vitalsigns
+  # Base class for commands
   class Command
-    def execute(command, options={})
-      Open3.popen3(command) do |stdin, stdout, stderr, wait_thr|
-        return stdout.read
+    def execute(command, _options = {})
+      Open3.popen3(command) do |_stdin, stdout, _stderr, _wait_thr|
+        return parse(stdout)
       end
     end
 
     def parse(output)
-      puts output
+      output.read
     end
   end
-end
\ No newline at end of file
+
+  def self.find_device(dire_path = '/var')
+    cmd = "df -h #{dire_path} | awk '{ print $1}'"
+    c = Vitalsigns::Command.new
+    line = c.execute(cmd)
```

Comment:
This can be `Vitalsigns::Command.new.execute(cmd)` unless variable `c` is re-used.